<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLineEdit" name="output_le"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="channel_le"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="axis_select_db">
       <item>
        <property name="text">
         <string>X Axis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y Axis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Temp</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="source_select_cb">
       <item>
        <property name="text">
         <string>Single-ended</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Differential</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDial" name="channel_selector_dl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
